### DISCUSSION TITLE
[Esame 9 Settembre 2019] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#229] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/11734b5f-8c35-48fa-bfe4-f46411b1ebf9)

**Comment by LeoCosta02** [comment]
identico al foglio 8 es 4
- #189
